.travis.yml

```diff
@@ -80,6 +81,11 @@ jobs:
       env: OPAMBUILDTEST=false OCAML_VERSION=4.04.2
 
     # === Build Tests ===
+    # Javascript (no tests)
+    - stage: "test"
+      name: "js (no tests)"
+      script: rsc/extra/check_js_test.sh
+      env: OPAMBUILDTEST=false OCAML_VERSION=4.09.0
```

Comment:
It would be better to put that test at the end, rather than at the start of the `test`stage (jobs are executed in the order specified iirc, and we'll likely want to test the regular builds first).

Reply:
I've put it at the beginning since it's only in dev/test phase, i'll of course put it at the end when i'ill have a working script.

Reply:
JS testing phase is now done at the end. Since this tests take a lot of time ~20 minutes, I'm only testing the js-node on the 4.09.0 version of OCaml. Do you think we should test more versions .

Reply:
Testing only on a single ocaml version should be fine.